src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@import '/styles/variables/variables';
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+
+    padding-inline: 3px;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
+    margin-top: 36px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
```

Comment:
Empty line before the opening of new brackets/declaration, please
This is valid for all following declarations